### PR TITLE
fix (ras-acc): update max-height for modal styles

### PR DESCRIPTION
**Pull request by laurelfulford**
### All Submissions:

* [ ] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/trunk/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR goes with https://github.com/Automattic/newspack-blocks/pull/1834; the testing steps can be found there.

### How to test the changes in this Pull Request:

1.
2.
3.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: assets/newspack-ui/scss/_modals.scss
Comment (chickenn00dle) [inline assets/newspack-ui/scss/_modals.scss:51]: Can we add a breakpoint for mobile devices so this change doesn't impact desktop?

**Review by chickenn00dle — CHANGES_REQUESTED** [review]

File: assets/newspack-ui/scss/_modals.scss
Comment (laurelfulford) [reply assets/newspack-ui/scss/_modals.scss:51]: We definitely can, but in my testing they seemed to appear the same -- are you noticing a big difference?

This is what I got when testing the epic/ras-acc branch on the left, and this branch on the right in Firefox:

image

Comment (chickenn00dle) [reply assets/newspack-ui/scss/_modals.scss:51]: Hmm. I retested and it does look like there isn't a difference after all. I might have unwittingly resized my browser a bit or something 🤷 . In any case, gonna go ahead and just approve this pending design review approval. Sorry for the mixup!

**Review by chickenn00dle — APPROVED** [review]

File: assets/newspack-ui/scss/_modals.scss
Comment (laurelfulford) [reply assets/newspack-ui/scss/_modals.scss:51]: No worries! Thanks for taking a second look 😄

**Pull request merged**
